frontend/src/components/Contexts/InstallationContext.tsx

```diff
@@ -23,16 +28,23 @@ interface Props {
 const defaultInstallation = {
     installationCode: '',
     installationName: '',
+    installationDecks: [],
+    installationAreas: [],
```

Comment:
I quite like this idea of having most things related to an installation in the installation context